api/apps/api/test/utils/scenario-calibration-repo.test.utils.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  CalibrationRunResult,
+  ScenarioCalibrationRepo,
+} from '../../src/modules/blm/values/scenario-calibration-repo';
+
+export class FakeScenarioCalibrationRepo implements ScenarioCalibrationRepo {
+  fakeCalibrationResults = [
```

Comment:
this is the change I mentioned in my earlier comment a few moments ago:

```suggestion
  fakeCalibrationResults: CalibrationRunResult[] = [
```
(this would not be enough - we'd need to either hardcode or generate a UUID for `scenarioId` in the mock data below - I'd say it would be functionally identical, so up to you -- if you want to go this route, in any case)

Reply:
yes, this works, thanks!